editor/script_editor_debugger.cpp

```diff
@@ -492,17 +492,19 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 			pinfo.usage = PropertyUsageFlags(int(prop[4]));
 			Variant var = prop[5];
 
-			String hint_string = pinfo.hint_string;
-			if (hint_string.begins_with("RES:") && hint_string != "RES:") {
-				String path = hint_string.substr(4, hint_string.length());
-				var = ResourceLoader::load(path);
-			}
-
 			if (is_new_object) {
 				//don't update.. it's the same, instead refresh
 				debugObj->prop_list.push_back(pinfo);
 			}
 
+			if (Variant::get_type_name(var.get_type()) == "String") {
```

Comment:
It would be cleaner to check `var.get_type() == Variant::STRING` IMO instead of checking the human readable name.

Reply:
Yeah thats better, fixed!